src/main/java/com/ecore/roles/web/rest/UsersRestController.java

```diff
@@ -0,0 +1,47 @@
+package com.ecore.roles.web.rest;
+
+import com.ecore.roles.service.UsersService;
+import com.ecore.roles.web.UsersApi;
+import com.ecore.roles.web.dto.UserDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static com.ecore.roles.web.dto.UserDto.fromModel;
+
+@RequiredArgsConstructor
```

Comment:
**Not a must**
For sure it helps with getting less code, but in my opinion this is a case where it's better have the constructor with the Autowired annotation. It makes the code more readable in spring beans. Once we have both standards in the current code, I suggest to go just with one of them as much as possible.